src/Bookings.js

```diff
@@ -1,18 +1,58 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
-  const search = searchVal => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    fetch(`https://temporary-cyf-react.onrender.com/`)
+      .then((res) => {
+        if (res.status === 200) {
+          console.log("Status we get back", res.status);
+          // we want to pass this data on parsed using the json method
+          return res.json();
+        }
+        if (res.status === 500) {
+          console.log("Status we get back", res.status);
+          // we want to cancel out of this and set the Error
+          setLoading(false);
+          throw setError("this api sucks");
+        }
+      })
+      .then((data) => {
+        // console.log("data", data);
+        setLoading(false);
+        setError(false);
+        setBookings(data);
+      })
+      .catch((error) => {
+        console.log("3wq ZV");
+      });
+  }, []);
+
+  const search = (searchVal) => {
     console.info("TO DO!", searchVal);
+    setBookings((prev) =>
```

Comment:
If we filter here based off the previous state value, then every search will have fewer and fewer options to actually grab. If I search and have no results, and then search again for someone who exists, then I wouldn't successfully find them.

To solve this, the most obvious option would be to store two arrays in state - listOfClients and filteredListOfClients maybe. We could simplify this futher though by just storing the actual search term in state as a string and then filtering the results based off of that.